Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Base/readme.md

```diff
@@ -0,0 +1,30 @@
+# MDTI-Base
+
+## Overview
+This playbook creates a shared Connection for all Microsoft Defender Threat Intelligence(MDTI) playbooks to leverage. This eases the configuration process for a user during deployment of the MDTI solution. In time, this base playbook may be extended to set more functionality. If you have trouble accessing your account or your credentials contact your account representative.
+
+## Pre-deployment Instructions
+
+1. Azure AD App Registration credentials(ClientId/ClientSecret/TenantId) with MDTI API Permissions are needed when configuring this playbook. Please check here for instructions on creating an App Registration [Azure Client App](https://learn.microsoft.com/en-us/rest/api/azure/#register-your-client-application-with-azure-ad) page. 
+
+MDTI API documentation for more details, [MDTI API](https://learn.microsoft.com/en-us/graph/api/resources/security-threatintelligence-overview?view=graph-rest-beta&branch=pr-en-us-20472)
```

Comment:
The Link for MDTI API Documentation is correct, it will go live on March 28th. you can validate the link by adding review in front of it. like this, (https://review.learn.microsoft.com/en-us/graph/api/resources/security-threatintelligence-overview?view=graph-rest-beta&branch=pr-en-us-20472)